test/e2e/helm/kuma_helm_deploy_global_zone.go

```diff
@@ -169,23 +169,22 @@ interCp:
 		})
 	})
 
-	It("should execute admin operations on Global CP", func() {
+	// Seems like admin operations are broken:
+	// error while running command: exit status 1; Error: Could not execute admin operation (could not send XDSConfigRequest: Resource not found: type="Dataplane" name="demo-client-84b6dc7cb8-wq44w.kuma-system" mesh="default")
```

Comment:
@lobkovilya I think this is a result of the rename. I've changed the tests but it now seems to be reaching the wrong DP (this test wasn't running anymore which is why you didn't catch it :( ).